### PR TITLE
Gen addresses - Tabs

### PR DESCRIPTION
**Pull request by jessiemongeon1**
Thank you for your contribution to the IC Developer Portal. This repo contains the content for https://internetcomputer.org and the ICP Developer Documentation, https://internetcomputer.org/docs/. 

If you are submitting a Pull Request for adding or changing content on the ICP Developer Documentation, please make sure that your contribution meets the following requirements:

- [x] Follows the [developer docs style guide](style-guide.md).
- [x] Follows the [best practices and guidelines](README.md#best-practices).
- [x] New documentation pages include [document tags](README.md#document-tags).
- [x] New documentation pages include [SEO keywords](README#seo-keywords).
- [x] New documents are in the `.mdx` file format to support the previous two components. 
- [x] New documents are registered in [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js), otherwise, it will not appear in the
  side navigation bar.
- [x] Make sure that the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file is
  filled with new documents that you added. This way we can ensure that future Pull Requests are reviewed by the right people.

### CODE REVIEW

File: docs/developer-docs/multi-chain/generating-addresses.mdx [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+
+:::caution
+
+Not sure if a chain is EVM-compatible? Check [ChainList](https://chainlist.org/), a full database of EVM-compatible chains.
+
+:::
+
+Wallet and contract addresses can be queried using the [EVM RPC canister](/docs/current/developer-docs/multi-chain/ethereum/evm-rpc/evm-rpc-canister).
+
+#### Generating an ETH address
```
Comment (jennifertrin) [inline docs/developer-docs/multi-chain/generating-addresses.mdx:99]: Should we note "Generating an ETH or EVM address"?

File: docs/developer-docs/multi-chain/generating-addresses.mdx [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+
+To create an ETH address for your canister, first you will need to obtain an ECDSA public key associated to your canister. You can get this key by calling the `ecdsa_public_key` API method of the management canister:
```
Comment (jennifertrin) [inline docs/developer-docs/multi-chain/generating-addresses.mdx:101]: Same here, "create an ETH or EVM address"

File: docs/developer-docs/multi-chain/generating-addresses.mdx (hunk 1 above)
Comment (jessiemongeon1) [reply docs/developer-docs/multi-chain/generating-addresses.mdx]: Revised

File: docs/developer-docs/multi-chain/generating-addresses.mdx (hunk 2 above)
Comment (jessiemongeon1) [reply docs/developer-docs/multi-chain/generating-addresses.mdx]: Revised

**Review by jennifertrin — APPROVED** [review]

**Pull request merged**